src/pages/en/tutorial/2-pages/1.md

```diff
@@ -133,7 +133,7 @@ When you are happy with the way your preview looks, and you want to publish your
 4. After waiting a few minutes, visit your Netlify URL to verify that your changes are published live.
 
 :::tip[commit and deploy regularly]
-Follow these steps every time you stop working! Your changes will be updated in your GitHub repository, and your Netlify website will be rebuilt and republished.
+Follow these steps every time you pause working! Your changes will be updated in your GitHub repository, and your Netlify website will be rebuilt and republished.
```

Comment:
```suggestion
Follow these steps every time you pause working! Your changes will be updated in your GitHub repository. If you've deployed to a Netlify website, it will be rebuilt and republished.
```
do you WANT to be more explicit here?

Reply:
yep, thanks!